go/install/install_osx.go

```diff
@@ -557,6 +566,75 @@ func InstallKBFS(context Context, binPath string, force bool, timeout time.Durat
 	return nil
 }
 
+// Return where the NativeMessaging host manifest lives on this platform.
+func kbnmManifestPath() string {
+	// Find path to write JSON into, one of:
+	// "/Library/Google/Chrome/NativeMessagingHosts"
+	// "$HOME/Library/Application Support/Google/Chrome/NativeMessagingHosts"
+	// XXX: Are we always root?
+	return "/Library/Google/Chrome/NativeMessagingHosts"
+}
+
+// kbnmHostName is the name of the NativeMessage host that the extension communicates with.
+const kbnmHostName = "io.keybase.kbnm"
+
+// kbnmDescription is the description of the purpose for the manifest whitelist.
+const kbnmDescription = "Keybase Native Messaging API"
+
+// kbnmExtensionID is the
+const kbnmExtensionID = "chrome-extension://bddkhaigcjacfohdeklflaigfnebicfm/"
+
+// InstallKBNM installs the Keybase NativeMessaging whitelist
+func InstallKBNM(context Context, binPath string, log Log) error {
+	// Find path of the keybase binary
+	hostPath, err := chooseBinPath(binPath)
+	if err != nil {
+		return err
+	}
+
+	// Host manifest, see: https://developer.chrome.com/extensions/nativeMessaging
+	hostManifest := struct {
+		Name           string   `json:"name"`
+		Description    string   `json:"description"`
+		Path           string   `json:"path"`
+		Type           string   `json:"type"`
+		AllowedOrigins []string `json:"allowed_origins"`
+	}{
+		Name:        kbnmHostName,
+		Description: kbnmDescription,
+		Path:        hostPath,
+		Type:        "stdio",
+		AllowedOrigins: []string{
+			kbnmExtensionID,
+		},
+	}
+
+	hostsPath := kbnmManifestPath()
+	jsonPath := filepath.Join(hostsPath, kbnmHostName+".json")
+
+	fp, err := os.OpenFile(jsonPath, os.O_WRONLY, 0444)
```

Comment:
0644 would work too. Actually now that I read it, I need `os.O_CREATE | os.O_WRONLY`.